### ISSUE TITLE
Todolist using Sqlite

### ISSUE THREAD

**Issue by aparnarsjain** [aparnarsjain/ToDoList#1]
/cc @codepathreview @codepath

Rewrote the app using sqlite for storage instead of text files. UI enhancements to follow.

**Comment by nesquena** [comment]
OK great, we'll keep an eye out for the UI enhancements (which I'm assuming is why there's no GIF in the README yet for this repo). I'd also encourage you to take a look at [this guide on code organization best practices](http://guides.codepath.com/android/Organizing-your-Source-Files) to cleanup your source code as well.

**Comment by aparnarsjain** [comment]
Yes, I didn't want to create a GIF yet, the app does not look pretty yet :). Thanks for the link to the guide. I'll organize the code as well.
